tests/service/Symbols.fs

```diff
@@ -396,4 +386,42 @@ let bytes = @"yo"B
 
         match getBindingExpressionValue parseResults with
         | Some (SynExpr.InterpolatedString(_,  kind, _)) -> kind |> should equal SynStringKind.Regular
-        | _ -> failwithf "Couldn't find const"
\ No newline at end of file
+        | _ -> failwithf "Couldn't find const"
+
+module SynModuleOrNamespace =
+    [<Test>]
+    let ``DeclaredNamespace range should start at namespace keyword`` () =
+        let parseResults = 
+            getParseResults
+                """namespace TypeEquality
+
+/// A type for witnessing type equality between 'a and 'b
+type Teq<'a, 'b>
+"""
+
+        match parseResults with
+        | Some (ParsedInput.ImplFile (ParsedImplFileInput (modules = [ SynModuleOrNamespace.SynModuleOrNamespace(kind = SynModuleOrNamespaceKind.DeclaredNamespace; range = r) ]))) ->
+            assertRange 1 0 4 8 r
```

Comment:
Could the start/end positions be tupled like `(startLine, startColumn) (endLine endColumn)`?

Reply:
Sure